tests/model_serving/model_server/inference_graph/conftest.py

```diff
@@ -34,28 +39,58 @@ def dog_breed_inference_graph(
             ],
         }
     }
+
+    annotations = {}
+    labels = {}
+    networking_label = Labels.Kserve.NETWORKING_KNATIVE_IO
+    try:
+        if request.param.get("deployment-mode"):
+            annotations[Annotations.KserveIo.DEPLOYMENT_MODE] = request.param["deployment-mode"]
+            if request.param["deployment-mode"] == KServeDeploymentType.RAW_DEPLOYMENT:
+                networking_label = Labels.Kserve.NETWORKING_KSERVE_IO
+    except AttributeError:
+        pass
+
+    try:
+        if request.param.get("enable-auth"):
+            annotations[Annotations.KserveAuth.SECURITY] = "true"
+    except AttributeError:
+        pass
+
+    try:
+        name = request.param["name"]
```

Comment:
Does the attribute error recognize empty strings? like " " (string with blank space), or special characters?

Reply:
It doesn't.
The `AttributeError` is here, because the `param` in  `request` can be undefined. Thus, trying to reference the undefined `request.param` throws an `AttributeError` exception.